pamelalozano/src/main/webapp/maps.js

```diff
@@ -13,7 +13,31 @@
 // limitations under the License.
 
 /** Creates a map according to the info of the indicator in the world's bank data */
-async function createMapChart(){
+
+document.getElementById("searchIndicatorBtn").addEventListener("click",(e) => {
+    document.getElementById("map").innerHTML="";
+    let  form = document.getElementById("selectIndicator");
+    let selected = form.options[form.selectedIndex];
+    let year = checkIndicatorYear(selected.value);
+    document.getElementById("indicatorTitle").innerText=selected.text+" - "+year;
+    createMapChart(selected.value, selected.text, year);
+})
+
+/* Returns the year with the last available information of each indicator*/
+function checkIndicatorYear(indicator) {
```

Comment:
A map might be a better than a switch statement for this lookup.

```js
const indicatorYear = {
  'ER.PTD.TOTL.ZS': 2018,
  ...
};
```

```js
let year = indicatorYear[selected.value];
```

Reply:
Okay thanks! Done :)